### ISSUE TITLE
extracting conda.tar.gz file in release v0.9

### ISSUE THREAD

**Issue by iahdb** [JordyCoolen/easyseq_covid19#8]
Hi Jordy! 

I'm struggling with extracting the conda envrionments for the v0.9 release. 

I wget the zipped file from the link provided, renamed to conda.tar.gz so I could extract as you've told me before. 

...But! -- I get "permission denied" when the extraction process tries to create new directories for all the new environments. 
[screenshot showing error messages] 
<img width="1400" alt="Screenshot 2022-05-06 at 16 43 42" src="https://user-images.githubusercontent.com/95859183/167167004-06802bce-f6a2-497a-93ea-5a841a5a4d8b.png">

Am I missing something? I don't remember this being an issue in the previous release. 

Thanks for your help! 🤓

**Comment by JordyCoolen** [comment]
Dear iahdb,

The conda.tar.gz file has not been changed with version 0.9.
It still contains an old version of Pangolin.

However, you should be able to extract it properly. Most likely you are extracting it on a location you don't have
full rights to create folders etc. Please try to extract it to another location.

Hopefully, that will work.

Kind regards,
Jordy

**Comment by iahdb** [comment]
Hi Jordy,

Thanks for you reply! 

I realised it was because I hadn't activated the Docker container - Oops! It worked when I activated the docker container.

I was extracting the conda file in the extracted "easyseq_covid19-0.9" project folder as detailed in the install instructions.  
Editing or moving files in the git repository always gives me a "permission denied" error when not in the Docker container. I'm not sure if that's normal, but everything works fine when I'm in the docker container so it's easily fixed. 

Best wishes,
I'ah
